### ISSUE TITLE
[Classic] v3.9.0 PlayerName and Invites module errors

### ISSUE THREAD

**Issue by Road-block** [sylvanaar/prat-3-0#160]
**Describe the bug**
Addon Spams errors on load

**To Reproduce**

**Expected behavior**

**Screenshots**
```
1x Prat-3.0-3.9.0\modules\PlayerNames.lua:1152: attempt to call field 'GuildRoster' (a nil value)
Prat-3.0-3.9.0\modules\PlayerNames.lua:1152: in function 'OnModuleEnable'
Prat-3.0-3.9.0\addon\modules.lua:189: in function <Prat-3.0\addon\modules.lua:178>
```
```
25x Prat-3.0-3.9.0\modules\PlayerNames.lua:1354: attempt to call field 'GuildRoster' (a nil value)
Prat-3.0-3.9.0\modules\PlayerNames.lua:1354: in function '?'
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua:119: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:119>
[C]:: ?
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua:29: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua:64: in function 'Fire'
Ace3\AceEvent-3.0\AceEvent-3.0-4.lua:120: in function <Ace3\AceEvent-3.0\AceEvent-3.0.lua:119>
```
```

3x Prat-3.0-3.9.0\modules\Invites.lua:393: attempt to call field 'CanInvite' (a nil value)
Prat-3.0-3.9.0\modules\Invites.lua:393: in function <Prat-3.0\modules\Invites.lua:386>
(tail call): ?
(tail call): ?
[C]:: in function 'gsub'
Prat-3.0-3.9.0\services\patterns.lua:197: in function 'MatchPatterns'
Prat-3.0-3.9.0\addon\addon.lua:695: in function <Prat-3.0\addon\addon.lua:613>
(tail call): ?
FrameXML\ChatFrame.lua:2797: in function 'ChatFrame_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>
```

**Version Information (please complete the following information):**
 - Wow Version: (current Classic) 1.13.5.36035
 - Prat Version: 3.9.0

**Comment by Road-block** [comment]
GuildRoster is not namespaced under C_GuildInfo in current  live Classic (maybe you tested on 1.13.6 PTR and they're transitioning to some 9.x stuff there?)

CanInvite similarly is not namespaced under C_PartyInfo, the previous CanGroupInvite needs to be used for live Classic.

I do not have all Prat modules enabled so it's likely there's more premature changes like those that I didn't catch.

**Comment by Foamzr** [comment]
Can confirm that I am getting the same as above.
Whenever I have a message that says 'inv' or 'invite' I get a LUA error, and it also does not show in my chat frame.

[LUA_Error_Prat.txt](https://github.com/sylvanaar/prat-3-0/files/5405111/LUA_Error_Prat.txt)

Prat 3.0-3.9.0

**Comment by samzorio** [comment]
Same thing here had to revert to previous version

**Comment by KyrosKrane** [comment]
This is fixed by #167
